contracts/test/LockedNORIV1Helper.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: MIT
```

Comment:
this file was renamed from LockedNORIHelper.sol to LockedNORIV1Helper.sol because now the two versions of the contracts differ in their interfaces to the point that V1 locked nori needs its own helper (this helper contract is used in the upgrade test).

`LockedNORIHelper.sol` is now up to date with the new version of `LockedNORI.sol` so it doesn't use`createGrant` and can instead create grants using `batchCreateGrants`

Reply:
do we still need tests for v1?

Reply:
this is used in the upgrade test - otherwise the tests apply to V2

Reply:
The upgrade test (and the V1 contract) could be removed now that the upgrade has happened but let's wait until the audit changes are done.